packages/swap/src/adapters/api.ts

```diff
@@ -5,28 +5,23 @@ import {
   asOpenswapTokenId,
   asOpenswapTokenIdHex,
   asYoroiBalanceTokens,
+  asYoroiOrders,
   // asYoroiOrders,
   asYoroiPools,
 } from './transformers'
 import {getTokensMock} from './tokens.mocks'
 import {getPoolsMock} from '../adapters/pools.mocks'
-import {getOrdersMock} from '../adapters/orders.mocks'
 
 export const makeSwapApi = (
   // FIX: network Yoroi type need to bring from the wallet // chain Id
-  {network /* , stakingKey */}: {network: 1 | 0 | 300; stakingKey: string},
+  {network, stakingKey}: {network: 1 | 0 | 300; stakingKey: string},
   deps?: {openswap?: OpenSwapApi},
 ): Readonly<Swap.Api> => {
   const api =
     deps?.openswap ?? new OpenSwapApi(network === 0 ? 'mainnet' : 'preprod')
 
-  const getOrders: Swap.Api['getOrders'] = async (): Promise<
-    Swap.OpenOrder[]
-  > => {
-    // return api.getOrders(stakingKey).then(asYoroiOrders)
-    await new Promise((r) => setTimeout(r, 500))
-
-    return getOrdersMock
+  const getOrders: Swap.Api['getOrders'] = async () => {
```

Comment:
completed orders is going to break can you update the injected  `getOrders` that is being used in the completed orders hook please?

thanks

Reply:
im working on it

Reply:
ah! you took that into account 👍🏻